src/app/(profile)/_actions/countProfileView.ts

```diff
@@ -0,0 +1,18 @@
+'use server'
+import {
+  getProfileById,
+  incrementProfileViewCountById,
+} from '@/backend/profile/profile.service'
+import { withSentry } from '@/utils/errHandling'
+
+export const countProfileView = withSentry(
+  async (profileId: string, userProfileId?: string) => {
+    const foundProfile = await getProfileById(profileId)
+
+    if (profileId === userProfileId) {
+      throw new Error("User can't incerement own profile view count")
```

Comment:
**issue:** We should not throw error here, because it's not an error but intended behaviour to not count own page view

Reply:
let's just call `const updatedProfile = await incrementProfileViewCountById(profileId)` when `profileId !== viewerProfileId`